operator/v1/types_clustermanager.go

```diff
@@ -339,3 +339,54 @@ type ClusterManagerList struct {
 	// Items is a list of deployment configurations for registration and work distribution controllers.
 	Items []ClusterManager `json:"items"`
 }
+
+const (
+	// The types of ClusterManager condition status.
+	// ConditionClusterManagerApplied is the ClusterManager condition status which means all components have been applied on the hub.
+	ConditionClusterManagerApplied = "Applied"
+	// ConditionHubRegistrationDegraded is the ClusterManager condition status which means the registration is not ready to server on the hub.
+	ConditionHubRegistrationDegraded = "HubRegistrationDegraded"
+	// ConditionHubPlacementDegraded is the ClusterManager condition status which means the placement is not ready to server on the hub.
+	ConditionHubPlacementDegraded = "HubPlacementDegraded"
+	// ConditionProgressing is the ClusterManager condition status which means all components are applying on the hub.
```

Comment:
it mostly means the cluster manager is in upgrading phase, or not fully started yet.